integration/init_test.go

```diff
@@ -109,4 +112,12 @@ func TestIntegration(t *testing.T) {
 	suite("OfflineApp", testOffline)
 	suite("SimpleApp", testSimpleApp)
 	suite.Run(t)
+
+	// Clean up jammy stack image
+	dockerClient, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	Expect(err).NotTo(HaveOccurred())
+	_, err = dockerClient.ImageRemove(gocontext.Background(), "paketobuildpacks/builder-jammy-buildpackless-base", types.ImageRemoveOptions{Force: true})
+	Expect(err).NotTo(HaveOccurred())
+	_, err = dockerClient.ImageRemove(gocontext.Background(), "paketobuildpacks/run-jammy-base:latest", types.ImageRemoveOptions{Force: true})
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
Can you not use the `occam` `docker.Image.Remove` method for these?

Reply:
Totally. No idea why I did it this way

Reply:
fixed